### PR TITLE
Increasing memory due to memoery limit error

### PR DESCRIPTION
**Pull request by pokearu**
*Description of changes:*
Bumping up memory due to memory insufficient error.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: jobs/aws/eks-anywhere-build-tooling/hook-presubmit.yaml [hunk 1]
```diff
@@ -54,10 +54,10 @@ presubmits:
           periodSeconds: 10
         resources:
           requests:
-            memory: "16Gi" # Higher memory for kernel builds
+            memory: "26Gi" # Higher memory for kernel builds
```
Comment (jaxesn) [inline jobs/aws/eks-anywhere-build-tooling/hook-presubmit.yaml:57]: Double check that this a valid instance configuration for fargate. Are these builds happening in buildkitd or on the main container?  You may want to add a resource sectional to the buildkitd container to make it easier to validate when added together it falls within a valid fargate range

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html#fargate-tasks-size

Comment (pokearu) [reply jobs/aws/eks-anywhere-build-tooling/hook-presubmit.yaml:57]: So I checked we had 26Gi for some of our EKS-D jobs, so i think its a valid?
Also the build is run using `buildkit`, so sounds we might need something over there.

Comment (jaxesn) [reply jobs/aws/eks-anywhere-build-tooling/hook-presubmit.yaml:57]: Yeah 4cpu and 26 mem should be ok, but I'm honestly not sure what happens when buildkit has no requests? I would drop 512 or so from the CPU and give it to buildkitd at least, and give it 1gb of men. If the bulk of the work for this build is actually happening in the buildkitd, like from a dockerfile, then you may actually want most of the resources on that container instead?

**Comment by pokearu** [comment]
Currently using a different approach to build.
New changes on #88

**Pull request closed without merging**